### PR TITLE
Doc-838 Create AA database on K8s

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Jira: DOC-838
Staging: https://docs.redis.com/staging/DOC-838/kubernetes/re-clusters/create-aa-database/

### CODE REVIEW

**Comment by laurentdroin** [comment]
For openshift:

> Verify API hostname works??????

It's the same as with ingress. Same CURL call. From outside k8s, we don't need to know whether ingress or route is used.

File: content/platforms/kubernetes/redb/create-aa-database.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  If the API call fails, create a DNS alias that resolves your API hostname (`<api-hostname>`) to the IP address for the ingress controller's LoadBalancer.
+
+### Using OpenShift routes
+
+1. Make sure your Redis Enterprise cluster has a unique name. By default, OpenShift always names the REC "`rec`". You'll need to manually rename any RECs using Active-Active with a unique name and reapply [scc.yaml](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/openshift/scc.yaml) to the namespace.
+    ```
+    oc adm policy add-scc-to-group redis-enterprise-scc  system:serviceaccounts:<namespace>
+    ```
+
+1. If your cluster uses OpenShift routes, add the following to the `spec` section of your Redis Enterprise cluster (REC0 resource file.
```
Comment (rrelledge) [inline content/platforms/kubernetes/redb/create-aa-database.md:117]:
```suggestion
1. If your cluster uses OpenShift routes, add the following to the `spec` section of your Redis Enterprise cluster (REC) resource file.
```

File: content/platforms/kubernetes/redb/create-aa-database.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+1. Verify you can access the API from outside the K8s cluster.
+
+    ```bash
+   curl -k -L -i -u <username>:<password> https://<api-hostname>/v1/
+    ```
+
+## Create an Active-Active database with `crdb-cli`
+
+The `crdb-cli` command can be run from any Redis Enterprise pod hosted on any participating K8s cluster. You'll need the values for the [required parameters]({{<relref "platforms/kubernetes/redb/create-aa-database.md#document-required-parameters" >}}) for each Redis Enterprise cluster.
```
Comment (rrelledge) [inline content/platforms/kubernetes/redb/create-aa-database.md:144]: The anchor wasn't working as expected.
```suggestion
The `crdb-cli` command can be run from any Redis Enterprise pod hosted on any participating K8s cluster. You'll need the values for the [required parameters]({{<relref "platforms/kubernetes/redb/create-aa-database#document-required-parameters" >}}) for each Redis Enterprise cluster.
```

**Review by rrelledge — APPROVED** [review]
Looks good so far. I just left a couple of small suggestions.

File: content/platforms/kubernetes/redb/create-aa-database.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+   curl -k -L -i -u <username>:<password> https://<api-hostname>/v1/
+    ```
+
+  If the API call fails, create a DNS alias that resolves your API hostname (`<api-hostname>`) to the IP address for the ingress controller's LoadBalancer.
+
+### Using OpenShift routes
+
+1. Make sure your Redis Enterprise cluster has a unique name. By default, OpenShift always names the REC "`rec`". You'll need to manually rename any RECs using Active-Active with a unique name and reapply [scc.yaml](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/openshift/scc.yaml) to the namespace.
+    ```
+    oc adm policy add-scc-to-group redis-enterprise-scc  system:serviceaccounts:<namespace>
```
Comment (kaitlynmichael) [inline content/platforms/kubernetes/redb/create-aa-database.md:114]:
```suggestion
    oc apply -f scc.yaml
    oc adm policy add-scc-to-group redis-enterprise-scc  system:serviceaccounts:<namespace>
```

File: content/kubernetes/re-clusters/create-aa-database.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+- **Replication hostname** `<replication-hostname>`:
+  - Description: Hostname used inside the ingress for the database
+  - Format: `<db-name><ingress-suffix>`
+  - Example value: `myaadb-cluster.ijk.redisdemo.com`
+  - How to get it: Combine `<db-name>` and `<ingress-suffix`> values you documented above.
+- **Replication endpoint** `<replication-endpoint>`:
+  - Description: Endpoint used externally to contact the database
+  - Format: `<db-name><ingress-suffix>:443`
+  - Example value: `myaadb-cluster.ijk.redisdemo.com:443`
+  - How to get it:`<replication-hostname>:443`"
```
Comment (kaitlynmichael) [inline content/kubernetes/re-clusters/create-aa-database.md:89]:
```suggestion
  - How to get it:`<replication-hostname>:443`
```

**Pull request merged**